### PR TITLE
[MINOR] fix: Correct the log file name for GC while using java11

### PR DESCRIPTION
**Pull request by xianjingfeng**
### What changes were proposed in this pull request?

Correct the log file name for GC. 

### Why are the changes needed?

Currently, the file name is `tags,time,uptime,level`.

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

Manual testing

### CODE REVIEW

**Comment by xianjingfeng** [comment]
@summaryzb @advancedxy PTAL

**Comment by rickyma** [comment]
I think this is similar to https://github.com/apache/incubator-uniffle/pull/1240. Could you merge this into your PR as well?

**Comment by xianjingfeng** [comment]
> I think this is similar to #1240. Could you merge this into your PR as well?

Sorry. I didn't found someone had create a same issue yesterday. I think we should merge #1240 , right? @rickyma @advancedxy

@qijiale76 please go ahead.

File: bin/start-coordinator.sh [hunk 1]
```diff
@@ -81,7 +81,7 @@ JVM_ARGS=" -server \
           -Xloggc:${RSS_LOG_DIR}/gc-%t.log"
 
 JAVA11_EXTRA_ARGS=" -XX:+IgnoreUnrecognizedVMOptions \
-          -Xlog:gc:tags,time,uptime,level"
+          -Xlog:gc=trace:file=${RSS_LOG_DIR}/gc-%t.log:tags,time,uptime,level:filecount=10,filesize=100m"
```
[the remark this answers is not in the document]
Comment (xianjingfeng) [reply bin/start-coordinator.sh:84]: 1. I have compared info and trace, and there is not much difference in the amount of logs, so I feel that trace will be better.
2. I did a simple test with java11+ and the gc logs can be printed normally.

**Comment by rickyma** [comment]
https://github.com/apache/incubator-uniffle/pull/1240 has been a while. I don't know the progress of it. Maybe you can communicate with @qijiale76. This is a useful PR, I think we need it ASAP.

**Pull request closed without merging**